bigbluebutton-config/bin/bbb-conf

```diff
@@ -718,8 +718,8 @@ if [[ $PORT_RANGE ]]; then
         sed -i "s/minPort=.*/minPort=$START_PORT/" /etc/kurento/modules/kurento/BaseRtpEndpoint.conf.ini
         sed -i "s/maxPort=.*/maxPort=$END_PORT/" /etc/kurento/modules/kurento/BaseRtpEndpoint.conf.ini
 
-        yq w -i /usr/local/bigbluebutton/bbb-webrtc-sfu/config/default.yml mediasoup.worker.rtcMinPort $START_PORT
-        yq w -i /usr/local/bigbluebutton/bbb-webrtc-sfu/config/default.yml mediasoup.worker.rtcMaxPort $END_PORT
+        yq w -i $WEBRTC_SFU_ETC_CONFIG mediasoup.worker.rtcMinPort $START_PORT
```

Comment:
bbb-install guarantees the /etc/.../production.yml file exists in 2.5+, but folks installing directly from packages might experience failures in this command due to the file/dir not existing (because it isn't generated by the packages by default).


Maybe we need a bbb-conf helper that guarantees the file exists (or alternatively make packaging/after-install generate an empty placeholder file when necessary)?

Reply:
How about if we just 'touch' the file in `bbb-conf --set-port-range` to ensure it exists?

I just pushed a one-line commit.